src/App/Fossa/PreflightChecks.hs

```diff
@@ -160,7 +160,7 @@ instance ToDiagnostic ProjectPermissionErr where
   renderDiagnostic CreateTeamProjectPermissionErr =
     Errata (Just projectPermissionErrHeader) [] $ Just "You do not have permission to create projects for the specified team"
   renderDiagnostic CreateProjectOnlyToTeamPermissionErr =
-    Errata (Just projectPermissionErrHeader) [] $ Just "You only have permission to create projects for your team"
+    Errata (Just projectPermissionErrHeader) [] $ Just "You only have permission to create projects for your team. Please specify a team in .fossa.yml or using the --team flag"
```

Comment:
I actually think this is more appropriate as part of the "help" text for this error: https://github.com/fossas/fossa-cli/blob/b4b091a89df5b9b6cf1d9f038fd28df855a8cdc8/src/App/Fossa/PreflightChecks.hs#L115

Can you incorporate its content there?

Reply:
Good call-out. Done! I also added periods to the Errata msgs, and `(s)` to `team` since you can be an admin/editor of multiple teams.